package.json

```diff
@@ -7,14 +7,18 @@
     "@testing-library/jest-dom": "^5.16.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "postcss-flexbugs-fixes": "^5.0.2",
+    "postcss-normalize": "^10.0.1",
+    "postcss-preset-env": "^7.0.2",
```

Comment:
- [ ] Please update the live link on the readme.

Reply:
Live link updated in README.MD